resource/login.php

```diff
@@ -0,0 +1,70 @@
+<?php
+    require 'vendor/autoload.php';
+
+    use Lcobucci\JWT\Builder;
+    use Lcobucci\JWT\JwtFacade;
+    use Lcobucci\JWT\Signer\Hmac\Sha256;
+
+    // Establish connection with database
+    require('database.php');
+    require('utils.php');
+    require('secret.php');
+
+    // Parse User object
+    $json = file_get_contents('php://input');
+    $data = json_decode($json);
```

Comment:
What happens if either one of these fails? This probably could happen if someone GET requested one of the URLs instead of POSTed them.

It isn't in the requirements and won't really affect the application but if you have the time a quick error handler might be nice in these circumstances.

Reply:
Fixed by https://github.com/ethandrzb/ContactManager/pull/1/commits/35d378c35b0f490ab4d4536d50b44798cbfb8365